src/components/Radio/styledRadio.tsx

```diff
@@ -33,9 +33,14 @@ export const StyledRadioRoot = styled.span<RadioWrapperProps>`
 
     &.knit-radio-checked {
       .knit-radio-inner {
-        &:after {
```

Comment:
`&:` also works but `&::` is better which clearly says that we are working with pseudo-elements, nice